src/Bicep.LangServer.IntegrationTests/SemanticTokenTests.cs

```diff
@@ -90,20 +91,31 @@ public async Task Correct_semantic_tokens_are_returned_for_params_file(string te
             });
 
             var lineStarts = TextCoordinateConverter.GetLineStarts(text);
-            var tokenSpans = CalculateTokenTextSpans(lineStarts, semanticTokens!.Data).ToArray();
+            var tokenInfos = CalculateSemanticTokenInfos(lineStarts, semanticTokens!.Data, helper).ToArray();
 
-            for (var i = 0; i < tokenSpans.Length; i++)
+            for (var i = 0; i < tokenInfos.Length; i++)
             {
-                TextSpan returnedSpan = tokenSpans[i];
+                TextSpan returnedSpan = tokenInfos[i].Span;
                 TextSpan expectedSpan = spans[i];
-
+    
                 returnedSpan.Position.Should().Be(expectedSpan.Position);
                 returnedSpan.Length.Should().Be(expectedSpan.Length);
+
+                string returnedType = tokenInfos[i].Type.ToString();
+                string expectedType = tokenTypeStrs[i];
+
+                returnedType.Should().Be(expectedType);
```

Comment:
To simplify, I am only comparing string values for types and not checking modifiers (as all modifier values for my sample input are "documentation."